hook.sh

```diff
@@ -39,7 +39,7 @@ if [ ! "$(type -P curl)" ]; then
     exit 5
 fi
 
-echo "Setting challenge to ${CERTBOT_VALIDATION} ..."
+echo -e "\e[32mSetting challenge to ${CERTBOT_VALIDATION} ...\e[0m"
```

Comment:
What is the purpose of this? Usually, color is used for errors or success messages, but not for regular progress printing. (certbot itself for example also does not use green color for their standard output.)

I would propose to keep stdout without color.

Reply:
Simply for visibility, to separate the hook's messages from certbot's. I'm not attached to it.

With the redirect to stderr (`>&2`), messages were being caught by certbot and not displayed until after the hook exited, in red, with an "error output" header. So the user would get a long delay with no output, and then a confusing dump of success messages that look like errors. Messages sent to stdout display in real time.

I can remove the escape codes, but I think we should still send status and success messages to stdout instead of stderr. Agreed?

Reply:
Yes, fully! My remark was only about the color :-)

Reply:
Also, I'm not attached to removing the coloring. I just wondered about the reason for it, because it looks like a success message before there is actual success.